rero_ils/modules/utils.py

```diff
@@ -1091,6 +1092,7 @@ def set_user_name(sender, user):
 
     user_name = None
     remove_user_name(sender, user)
+    set_user_data = {"id": user.id, "email": user.email}
```

Comment:
User email is not useful and might be a security concern to put in the logs.

Reply:
More useful information to debug and reproduce would be : `patron.pid` `patron.patron.barcode[0]` and the patron's organisation.